### ISSUE TITLE
[Question] make customer vie group dashboard puplic

### ISSUE THREAD

**Issue by Mirjka** [thingsboard/thingsboard#4308]
**Component**

 * Rule Engine

**Description**
Hi, I like to organize dashboards and devices in Customer Groups.
Is there a possibility to make a puplic dashboard / puplic device? I can´t find the "make puplic" button and don´t find a workaround.

Cheers!

**Environment**

 * OS:  Ubuntu 18
 * ThingsBoard: Professional (Version...have to ask our admin...)
 * Browser: FireFox 78.9.0esr

**Comment by vvlladd28** [comment]
HI @Mirjka,
In TB PE, you can't do make a public dashboard because TB PE supports only the public dashboard group/device group. In your case, your need to create a new dashboard group and added a dashboard that you want to make public, and then make this group public.

**Comment by Mirjka** [comment]
Hi @ashvayka,
thanks for youre advice. In that case, I have to organize the devices also outside the customer group to make them puplic aswell, don´t I?

**Comment by vvlladd28** [comment]
Hi @Mirjka,
No, you can organize a group of devices at the client and make them public

**Comment by Mirjka** [comment]
Cool, I got it. 

The way I solved it: After a added the customer group and a customer group user, i switch to that user. Within this profile It´s possible to add Device groups and Dashboard Groups and configure them as puplic. When I switch back to the customer group admin account, booth Devices and Dasboards are sorted in the costumer group and are marked as puplic aswell. So Iam able to creat customer groups with puplic and non-puplic dashboards, which are not able to see devices and dashboards of other customergroups  within the same tenant.

Cheers
